test/fixtures/variables-config.yml

```diff
@@ -0,0 +1,5 @@
+variables:
```

Comment:
Looks like this file is not used. Can we delete it?

Reply:
done